tests/admin/test_view.py

```diff
@@ -313,6 +313,25 @@ def test_post_existing_record(self):
         self._test_post_existing_record_response(response)
         assert len(EnterpriseCustomerUser.objects.filter(user_id=user.id)) == 1
 
+    def test_post_one_existing_one_new_record(self):
+        # precondition checks:
```

Comment:
Please add a docstring describing the use case(s) being covered by this test method.